src/app/shared/services/layout.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LayoutService {
+  // Service allowing modifications of the layout
+
+  private leftMenuAndHeadersDisplayed = new BehaviorSubject<boolean>(true);
+  leftMenuAndHeadersDisplayed$ = this.leftMenuAndHeadersDisplayed.asObservable().pipe(delay(0));
+
+  private withTask = new BehaviorSubject<boolean>(false);
+  withTask$ = this.withTask.asObservable().pipe(delay(0));
```

Comment:
Do you use `.pipe(delay(0))` because it was required or because you've copied it for another place?

Reply:
Yes.